### ISSUE TITLE
[Quest][WoTLK][Zul'Drak] Rampage Mini Event

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4650]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The quest it self works. It is just missing the mini event as i can see.
**Expected behaviour**: Tell us what should happen instead
Once you release Akali he should go on a rampgae killing the trolls that run in.
**Steps to reproduce the problem**:

1. Unlock all 4 chains 
2. No mini event
3. 

**Include proofs for this behaviour**
[Video of quest](https://www.youtube.com/watch?v=cfKUxE8xFAw)
![wowscrnshot_032917_182919](https://cloud.githubusercontent.com/assets/12252441/24479627/26be840e-14ae-11e7-8590-26a68da4a108.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
[Quest](http://www.wowhead.com/quest=12721/rampage)
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TheRealestMcCoy** [comment]
@TeridanDW This may be another one to sniff....

**Comment by TeridanDW** [comment]
Yes....... about this quest...... it's at the end of the quest chain of the zone, it will take me hours to get to it, however the event is indeed impresive. Luckily i found a person doing it and i could observe, sadly again without sniffer working (outdated).
Interesting fact - the stat squish actually broke this event as Akali 1 shots almost all adds with his charge.
Video from retail legion from 30.03.2017:
https://www.youtube.com/watch?v=XfapX7kMvOg&feature=youtu.be
And one more video from wotlk retail from 2008:
https://www.youtube.com/watch?v=po3h51bcuao

**Comment by TeridanDW** [comment]
Actually this event is working on PTR, however we should improve it a bit by:
 - when the extra adds spawn at the beggining of the event, they move near akali and after few secs they evade, however should should engage akali on spawn.
 - corpse despawn should be faster
 - respawn time of the event should be faster (check video above)

**Comment by Rushor** [comment]
i already have a fix for the combat fight - despawn/respawn still needs to be improved

**Comment by Rushor** [comment]
the event gets stucked if we want to execute it a second time after respawn

**Comment by Rushor** [comment]
DB/Quest: Zul'Drak - Rampage - Several Improvements

* set Akali active so the event does not stuck if a player earlier removes the area
* respawn all important trigger creatures on respawn of Akali, so the event can be done several times
* reduce respawn, corpsedelay and despawntime of Akali to 15 seconds
* fixup movejump action of Prophet of Akali
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4650
* commit: 234bf5372dd89a8ee5d23b83b7ca146a699a70d4

![wow_2017-03-30_13-07-19](https://cloud.githubusercontent.com/assets/5568820/24502178/f6160332-154c-11e7-95b9-c8e255dba96e.png)
![wow_2017-03-30_13-07-57](https://cloud.githubusercontent.com/assets/5568820/24502177/f60ba78e-154c-11e7-8bd4-a955b5ae2369.png)
![wow_2017-03-30_13-08-07](https://cloud.githubusercontent.com/assets/5568820/24502179/f61bad96-154c-11e7-951a-5b25aaa2925a.png)
![wow_2017-03-30_13-08-19](https://cloud.githubusercontent.com/assets/5568820/24502181/f629c2be-154c-11e7-9765-b48d5ae310d3.png)
![wow_2017-03-30_13-08-34](https://cloud.githubusercontent.com/assets/5568820/24502180/f62162ea-154c-11e7-9b68-b41f33e6ce02.png)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Hello, after few tests on PTR:

>  - set Akali active so the event does not stuck if a player earlier removes the area

Doesn't seems to work as I was able to get the event stucked by not been in the area while Akali was suppose to die at the end of the event.

This is what happened:
![image](https://user-images.githubusercontent.com/6797611/31930928-20a95918-b8aa-11e7-8e74-164c2db14c95.png)

 - Event started even though this chain is not released, should not happen:

![image](https://user-images.githubusercontent.com/6797611/31930998-5c5fd32e-b8aa-11e7-93f9-77188b3a1b88.png)

**Comment by Rushor** [comment]
related for active flag https://github.com/dalaranwow/dalaran-wow/issues/4219

**Comment by Rushor** [comment]
* set all summoned and affected creatures for the quest active with CREATURE_FLAG_EXTRA_ACTIVE_ON_SPAWN
